code/testapp/build.gradle

```diff
@@ -46,21 +46,17 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
 
     implementation project(":analytics")
-
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
-//    implementation('com.adobe.marketing.mobile:analytics:2.0.0-SNAPSHOT'){
+    //    implementation('com.adobe.marketing.mobile:analytics:2.0.0-SNAPSHOT'){
 //        transitive = false
 //    }
-    implementation('com.github.adobe.aepsdk-core-android:lifecycle:dev-v2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-    implementation('com.github.adobe.aepsdk-core-android:identity:dev-v2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-    implementation('com.github.adobe:aepsdk-assurance-android:2.0.0-alpha.1'){
-        transitive = false
-    }
 
+    implementation 'com.adobe.marketing.mobile:core:2.0.0'
+    implementation 'com.adobe.marketing.mobile:lifecycle:2.0.0'
+    implementation 'com.adobe.marketing.mobile:identity:2.0.0'
+    implementation 'com.adobe.marketing.mobile:assurance:2.0.0'
```

Comment:
Can you set each of these to "2.+"?

Reply:
I don't think we should use `+`, the reason has been posted above.

Reply:
can you revert this change after merge with dev?